include/zephyr/usb/class/usb_audio.h

```diff
@@ -100,39 +100,81 @@ enum usb_audio_fucs {
  * Refer to Table 2-1 - Table 2-4 from termt10.pdf
  */
 enum usb_audio_terminal_types {
-	/* USB Terminal Types */
+	/**
+	 * @name USB Terminal Types
+	 * @{
```

Comment:
Can you please explain to me why the same text as above is needed here?
Headers are files I work with, not the generated documentation, I really do not want the header to look uglier than necessary.

Reply:
If you look at the diff, I didn't add something new, but turned the `	/* USB Terminal Types */` comment into a proper Doxygen heading so that USB Terminal types,  Input terminal types, etc. also show up as grouped in the doxygen doc. Thanks!